### DISCUSSION TITLE
클라이언트/백엔드/알고리즘 개발 용어를 통일하는게 어떨까요?

### DISCUSSION THREAD

**Discussion by gitchannn** [sunnybraille/sunnybraille-server#103] [General]
당장 급하게 적용은 안될지라도 API endpoint를 최우선으로 시작해서, 점차 개발 용어를 통일해 나가면 좋을 것 같습니다

1. 되도록이면 코드 내에 pdf, png, ppt 등등 특정 파일 형태에 대한 변수들은 제거하겠습니다. 차차 확장해 나갈텐데 정말 딱 그 로직에 특수한 부분이 아니라면 확장성을 해치게 될 네이밍!
2. `점역`이라는 의미는 TRANSCRIBE로 통일하겠습니다. transbraille(x) translation(x) 저는 이거 통일하려면 지금 엔드포인트에서 `translations/`라는 말을 엄청나게 사용하기 땜시 엔드포인트를 조금 수정할 필요가 있을 것 같으니 그때 다시 api 문서 변경하면서 알려줄게요 @JunhyeongPark-kr 
3. braille이라는 말은 직접 사용해도 되는걸로 합시다! 어차피 이건 우리 서비스의 핵심이자 변할 일이 없는 말인 것 같아서요!
4. OCR 작업을 코드에 OCR이라고 박아놓는 거에 대해 어떻게 생각하시나요? 저는 썩 맘에들지 않긴 하지만, 그래도 ocr이라는 단어가 대체될 일은 비교적 적은 것 같아서 괜찮은 것 같습니다. 

cc. @J-nowcow @JunhyeongPark-kr @dbtjd7

**Comment by jun-brro** [comment]
모든 내용 잘 확인했습니다

4번의 경우, 코드에서는 OCR이라고 해도 될 것 같습니다. 혹시 제안하시는 용어가 있을까요?

**Reply by gitchannn** [reply]
맘에 쏙 들진 않지만 더 좋은게 없는 것 같아서 그냥 OCR로 갑시당

**Comment by J-nowcow** [comment]
코드에서 대소문자 구분을 어떻게 하고 계신지 궁금합니다. 언어별로 표준 포맷이 다를 것 같은데, 함수 / 변수 / 클래스 작명할 때 어떤 스타일로 작명하시는지 궁금해요

**Reply by gitchannn** [reply]
나는 서버 코드에서는 camelCase로 작성중이고, 데이터베이스에서는 snake_case로 작성중이야! 이거는 언어별로 다 달라서 이건 통일하기는 어려울지도 모르겠네!

**Comment by J-nowcow** [comment]
내용은 잘 확인했습니다 반영해서 개발하도록 하겠습니다!
